pkgs/by-name/qi/qidi-slicer-bin/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  webkitgtk,
```

Comment:
Please specify the version explicitly.

Reply:
An input like `webkitgtk_4_1` doesn't work in my case, I get the error
```console
/nix/store/c0djl96jgj1xmkag8pwp0lb9c4pifwvn-qidi-slicer-1.1.5-extracted/usr/bin/bin/qidi-slicer: error while loading shared libraries: libwebkit2gtk-4.0.so.37: cannot open shared object file: No such file or directory
```
at runtime. The "plain" `webkitgtk` input has version `2.44.3+abi=4.0`, which works.

Reply:
Use webkitgtk_4_0?

Reply:
Mea culpa, my branch was so out of date I didn't have that specific name yet. Rebasing solved the issue.